src/include/wiredtiger.h.in

```diff
@@ -53,6 +53,17 @@ typedef int64_t wt_off_t;
 #define WT_ATTRIBUTE_LIBRARY_VISIBLE    __attribute__((visibility("default")))
 #endif
 
+#if defined(_MSC_VER)
+#define WT_DIAGNOSTIC_PUSH      __pragma(warning(push))
```

Comment:
Creating a macro just to call `__pragma(warning(push))` seems overkill. Can we just disable the warning about the script on all machines?

Reply:
We need to disable the warning in the code. We deliver only source files to mongodb, therefore our build configuration (in CMake files) is not visible for them.

Using push/pop idiom is very common:
```c
_Pragma("clang diagnostic push")
_Pragma("clang diagnostic <disable something>")
_Pragma("clang diagnostic pop")
```